src/components/FloatingDropdown/FloatingDropdown.driver.ts

```diff
@@ -14,6 +14,7 @@ export interface FloatingDropdownDriver extends BaseUniDriver {
   clickOnDropdownBase(): Promise<void>;
   selectOptionAt(i: number): Promise<void>;
   getBaseSelectedValue(): Promise<string>;
+  isAriaExpanded(): Promise<string>;
```

Comment:
I think it is better to return `Promise<boolean>`

Reply:
changed from **isAriaExpanded** to **getAriaExpanded** and kept it as a string since I would like to know the value of the attribute. :)